app/main.py

```diff
@@ -2,0 +1,25 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
In the function definition, it would be beneficial to specify the expected type of the elements within the `friends` list for better readability and understanding of the code.